test/protocol.spec.ts

```diff
@@ -57,8 +57,8 @@ describe('protocol-contract', async () => {
 
     await evmIncreaseTimeForQueries(d.web3, 3);
 
-    await expectRejected(d.protocol.setProtocolVersion(DEPLOYMENT_SUBSET_MAIN, 3, currTime + 3, {from: d.functionalOwner.address}));
-    await expectRejected(d.protocol.setProtocolVersion(DEPLOYMENT_SUBSET_MAIN, 3, currTime + 2, {from: d.functionalOwner.address}));
+    await expectRejected(d.protocol.setProtocolVersion(DEPLOYMENT_SUBSET_MAIN, 3, currTime + 3, {from: d.functionalOwner.address}), /a protocol update can only be scheduled for the future/);
```

Comment:
The set is done for currTime + 3, why is it in the future?

Reply:
`curTime` at this point is from about 3 seconds ago - see line 58. I'll make this clearer